### PR TITLE
bump query parser

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: backend/poetry.lock [hunk 1]
```diff
@@ -2667,7 +2667,7 @@ version = "2.8.4"
 [[package]]
 category = "main"
 description = "Run a subprocess in a pseudo terminal"
-marker = "python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\""
+marker = "python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\" or python_version >= \"3.4\" and sys_platform != \"win32\" and (python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\")"
```
Comment (monaawi) [inline backend/poetry.lock:2670]: hm, it's odd it is adding this for you. It doesn't add it when I run ```poetry update```. What version of poetry are you using?

**Comment by monaawi** [comment]
Also, can you change the commit message to be ```setup: bump inspire...``` and add the JIRA task in the message as well?
This is typically the format we use for commit messages:
{area changed} : {description of the change}
* {JIRA TASK}
* {any needed operations like migrating, reindexing etc.}
* {extra info}

File: backend/poetry.lock (hunk 1 above)
Comment (MJedr) [reply backend/poetry.lock:2670]: I use poetry 1.0.5 and ```poetry update query-parser``` was the only command I run.

Comment (monaawi) [reply backend/poetry.lock:2670]: Can you try with 1.0.0?

**Pull request merged**